base/linalg/triangular.jl

```diff
@@ -1087,15 +1174,116 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
          1.276404810806775e-001,
          2.060962623452836e-001,
          2.879093714241194e-001]
+
```

Comment:
Not sure it is practical to compute them each time (actually I'm pretty sure it isn't), maybe we could tabulate them for different precision framework.